src/routes/error-router.ts

```diff
@@ -76,7 +76,7 @@ ErrorRouter.use((err: Error, req: Request, res: Response, next: NextFunction) =>
 
 	const errorStatusCode = errorCodes[err.message] || 500;
 	const message = messages[err.message];
-	const gitHubProduct = getCloudOrServerFromGitHubAppId(res.locals.gitHubAppId);
+	const gitHubProduct = getCloudOrServerFromGitHubAppId((res.locals.gitHubAppConfig || {}).gitHubAppId);
```

Comment:
Find it weird to grab the github product at this line. Anyway do a quick fix so that we can completely remove the `res.locals.gitHubAppId`, and use `res.locals.gitHubAppConfig.gitHubAppId`.